components/range_cache_memory_engine/src/range_manager.rs

```diff
@@ -262,6 +262,8 @@ pub struct RegionMetaMap {
     regions: HashMap<u64, CacheRegionMeta>,
     // active flag, cloned from RegionManager,
     is_active: Arc<AtomicBool>,
+
+    manual_load_range: HashSet<CacheRegion>,
```

Comment:
I don't see the benefit of using a `HashMap` here. Better replace it with `Vec` all the operations just iterate all the elements, or you can consider replace it with a `BTreeMap` like we do in `regions_by_range` as it provides better search performance.

Reply:
I'll use `Vec`. as impl `PartialOrd` and `Ord` for `CacheRegion` seems odd to me.